lib/killbill/helpers/active_merchant/utils.rb

```diff
@@ -1,3 +1,6 @@
+require 'thread'
+require 'thread_safe'
```

Comment:
Should we add an explicit dependency in the `gemspec`? It looks like it's a transitive dependency of `activesupport` and `tzinfo`, but it may be good to control the version explicitly?